File: programmers/Lv_1/KimHyunseo/완주하지못한선수.cpp
#include <string>
#include <vector>
#include<iostream>
#include <algorithm>

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
	string answer = "";
	sort(participant.begin(), participant.end());
	sort(completion.begin(), completion.end());
	for (int i = 0; i < completion.size(); i++)
	{
		if (participant[i].compare(completion[i]) != 0) {
			answer = participant[i];
			break;
		}
		else
		{
			answer = participant[completion.size()];
		}
	}
	return answer;
}
int main()
{
	vector<string> parti = { "mislav", "stanko", "mislav", "ana"};
	vector<string> comple= {"stanko", "ana", "mislav" };
	cout<<solution(parti,comple);
}

File: programmers/Lv_1/HwangJunghoon/완주하지 못한 선수.py
from collections import Counter

def solution(participant, completion):
    answer = Counter(participant) - Counter(completion)
    # Counter.collections 는 key값에 대해 빈도수를 value값으로 표시
    return list(answer.keys())[0]
    # key값 목록을 list로 변환 후 index를 사용하여 return